src/api/system/system.ts

```diff
@@ -80,6 +80,15 @@ export default class System extends Base {
     }
 
     /**
+     * Exits the Runtime.
+     * @return {Promise.<void>}
+     * @tutorial System.exit
+     */
+    public exit(): Promise<void> {
+        return this.wire.sendAction('exit-desktop', Object.assign({})).then(() => undefined);
```

Comment:
• Why `Object.assign({})` here?
• Why `.then(() => undefined);` here?

Reply:
I thought `this.wire` needed an Object, I just given is an empty Object not `Object.assign({})`

Reply:
Also I wrote `.then(() => undefined)` becasue in the javascript-adapter it does not recevice any args/input.